getsolus/packages
# gem: Update to 0.12.0

**Summary**

- Full changelog [here](https://gem.tuxfamily.org/changelog.html)
- Install appstream metadata (Part of getsolus/packages#1389)

**Test Plan**

- Open the app
- Set up emulator and rom databases
- Verify metadata with `appstream-builder --packages-dir=. --include-failed -v`

**Checklist**


**Notes**

This update broke database compability with previous version. Unfortunately user have to set up their game library again.
